### DISCUSSION TITLE
Mail de validation non reçu

### DISCUSSION THREAD

**Discussion by Pamplemousse9** [community/community#50616] [GitHub Education] [answered]
Bonjour, 

Suite à l'approbation de ma demande sur Github, je n'ai pas reçu le mail de confirmation contenant le mail avec le lien d'activation de la licence Canva PRO.

J'aurais souhaité savoir ce qu'il était possible de faire pour obtenir ce lien d'activation.

Vous remerciant par avance pour votre aide,

**Comment by samus-aran** [comment, marked as answer]
Hi @Pamplemousse9  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers). For students who have already claimed for your 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
